src/app.ts

```diff
@@ -4,15 +4,19 @@ import {
     EntityOptions,
     OrchestrationHandler,
     OrchestrationOptions,
+    RegisteredActivity,
+    RegisteredOrchestration,
```

Comment:
Yeah I hear you. I went with `RegisteredActivity` and `RegisteredOrchestration` because those were the names that were agreed upon on a different but similar PR. If you have different suggestions for these names, please let me know!

Reply:
Not at this time, I like these names at this time because they're simple, but at some point I think we should take a global look at the names we've been introducing to see if we need to document their differences somewhere.

Reply:
Sounds good :)